util-kotlin/src/main/kotlin/team/duckie/app/android/util/kotlin/exception.kt

```diff
@@ -18,40 +18,70 @@ package team.duckie.app.android.util.kotlin
  */
 sealed class DuckieException(code: String) : IllegalStateException("DuckieException: $code")
 
+/** API 에러로 인지되는 임계점. 아래 선언 값은 에러로 포함된다. */
+const val ApiErrorThreshold = 400
+private const val BadRequestCode = 400
+private const val UnAuthorizedCode = 401
+private const val ForbiddenCode = 403
+private const val NotFoundCode = 404
+private const val ConflictCode = 409
+private const val InternalServerCode = 500
+
+/** 서버 응답 StatusCode */
+@AllowMagicNumber("Server statusCode 구분 값 MagicNumber 허용해야 함")
+enum class DuckieStatusCode {
```

Comment:
```suggestion
enum class DuckieStatusCode(
   val code: Int,
 ) {
```
enum class에 status code를 포함시키면 위 코드들을 제거할 수 있을 것 같아요.